parsl/executors/high_throughput/executor.py

```diff
@@ -174,6 +174,16 @@ class HighThroughputExecutor(BlockProviderExecutor, RepresentationMixin):
 
     worker_logdir_root : string
         In case of a remote file system, specify the path to where logs will be kept.
+
+    enable_mpi_mode: bool
+        If enabled, MPI launch prefixes will be composed for the batch scheduler based on
+        the nodes available in each batch job and the resource_specification dict passed
+        from the app
+
+    mpi_launcher: str
```

Comment:
from a UX perspective, this is going to need some more serious documentation about why this is somehow different from the `Launcher` classes specified elsewhere in the htex config.

Reply:
I agree. Have you had a chance to look at the docs? Is there anything that could be added there for clarity?